fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisManager.java

```diff
@@ -206,6 +207,9 @@ public List<AnalysisInfo> buildAnalysisInfosForDB(DatabaseIf<TableIf> db, Analyz
                 if (table instanceof View) {
                     continue;
                 }
+                if (table.isTemporary()) {
```

Comment:
not analyze temporary table maybe not a good idea

Reply:
done